syntax = "proto3";

package core_system.customer.v1;

import "messages/common/request.proto";
import "messages/customer/response.proto";

option py_generic_services = true;

// CustomerService 정의
service CustomerService {
    rpc GetCustomersByShop(core_system.common.v1.GetByShopRequest) returns (GetCustomersByShopResponse);
}
